### PR TITLE
Buff the agent card removing the single use element.

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Buffed the agent card by removing the limit of being able to set your ID name, assingment, colour once. Now you can do change it indefinetely.

With this changed, the price of an agent card was raised to 2 and old agent card behaviour is kept the same for the ones nukies get as well as agent card found in the spec ops crate.

## Why's this needed? 
Underused item and frankly the limitation felt pointless and only hampered the use of the item.

## Changelog

```changelog
(u)Carbadox
(+)Buffed traitor PDA agent cards. You can now set your assignment, colour and name however many times you want. This change does not apply to the agent cards that Nukies get or the one found in the Spec Ops crate from QM.
```

### CODE REVIEW

**Comment by MatheusIsNotAvailable** [comment]
I would rather this be a different item, like an agent card deluxe or something, I like being able to get agent cards as they are now for 1 TC

File: code/obj/item/card.dm [hunk 1]
```diff
@@ -324,6 +325,10 @@ GAUNTLET CARDS
 	input = jointext(namecheck, " ")
 	return input
 
+/obj/item/card/id/syndicate/reusable
+	name = "agent card"
```
Comment (MarkNais) [inline code/obj/item/card.dm:329]:
```suggestion
	name = "reusable agent card"
```

May as well make it explicit.

**Review by MarkNais — CHANGES_REQUESTED** [review]
I'm not a "balance" guy, so this is just code-review on my part.

**Review by MarkNais — APPROVED** [review]
Ignoring balance: code seems fine.

**Comment by ThatFiveGuys** [comment]
Echoing Matheius's claim, rn agent cards are a good "I only have 1 TC purchase left and this will help" purchase. Making em 3 TC would kinda ruin that niche.

**Comment by flappybatpal** [comment]
I vote 2TC. The name functionality is a bit so-what when traitors usually get access to an identity computer.

**Comment by Carbadox** [comment]
Lowered the price to 2 due to feedback

**Comment by Bram-KEP** [comment]
Spec Ops crate one should share the effect imo as it makes sense and would incentivise using it as the crate is rarely used.

**Comment by PrimeNumb** [comment]
Definitely make this a separate item. Agent cards (as they are currently) are in a really good spot balance-wise.

**Pull request closed without merging**